xclim/indices.py

```diff
@@ -265,9 +264,23 @@ def daily_temperature_range(tasmax, tasmin, freq='YS'):
 
 
 @valid_daily_mean_temperature
-def freshet_start(tas, thresh=0, window=5, freq='YS'):
-    """Return first day of year when a temperature threshold is exceeded
+def freshet_start(tas, thresh=0.0, window=5, freq='YS'):
+    r"""First day of consistent threshold temperature.
+
+    Return first day of year when a temperature threshold is exceeded
     over a given number of days.
+
+    Parameters
+    ----------
+    tas : xarray.DataArray
+      Mean daily temperature
```

Comment:
I'm coming across function asking for deg C and others demanding K. Just for my reference, xclim performs the conversion handling for both, yes?

Reply:
@huard